src/mrinufft/trajectories/utils.py

```diff
@@ -173,69 +177,220 @@ class VDSpdf(StrEnum):
+
+    Notes
+    -----
+    The `Acquisition` class encapsulates the parameters needed for MRI acquisition,
+    including the field of view, image size, hardware specifications, and gyromagnetic
+    ratio.
+
+    It is designed to be used in conjunction with the `Hardware` class, which defines
+    the hardware constraints such as maximum gradient amplitude and slew rate.
+    The `default` class variable holds the default acquisition configuration, which can
+    be set using the `set_default` method. This allows for easy access to a standard
+    acquisition configuration without needing to instantiate a new `Acquisition` object
+    each time.
+
+    """
+
+    default: ClassVar[Acquisition]
+
+    fov: tuple[float, float, float]  # Field of View in m
+    img_size: tuple[int, int, int]  # Image size in pixels
+    hardware: Hardware = SIEMENS.TERRA  # Hardware configuration
+    gamma: Gammas = Gammas.HYDROGEN  # Hz/T
+    oversampling: int = 1  # Oversampling factor for the ADC
+    norm_factor: float = 0.5
+
+    def set_default(self) -> Acquisition:
+        """Make the current acquisition configuration the default."""
+        Acquisition.default = self
+        return self
+
+    def __getattr__(self, name):
+        """Allow access to hardware attributes directly from Acquisition."""
+        return getattr(self.hardware, name)
+
+    @property
+    def res(self) -> tuple[float, ...]:
+        """Resolution in meters."""
+        return tuple(fov / size for fov, size in zip(self.fov, self.img_size))
+
+
+# Create a default acquisition.
+Acquisition.default = Acquisition(
+    fov=(0.256, 0.256, 0.256), img_size=(256, 256, 256), hardware=Hardware()
+)
+
 ###############
 # CONSTRAINTS #
 ###############
 
 
 def normalize_trajectory(
     trajectory: NDArray,
-    norm_factor: float = KMAX,
-    resolution: float | NDArray = DEFAULT_RESOLUTION,
+    acq: Acquisition | None = None,
 ) -> NDArray:
     """Normalize an un-normalized/natural trajectory for NUFFT use.
 
     Parameters
     ----------
     trajectory : NDArray
         Un-normalized trajectory consisting of k-space coordinates in 2D or 3D.
-    norm_factor : float, optional
-        Trajectory normalization factor, by default KMAX.
-    resolution : float, np.array, optional
-        Resolution of MR image in meters, isotropic as `int`
-        or anisotropic as `np.array`.
-        The default is DEFAULT_RESOLUTION.
+    acq : Acquisition, optional
+        Acquisition configuration to use.
+        If `None`, the default acquisition is used.
 
     Returns
     -------
     trajectory : NDArray
         Normalized trajectory corresponding to `trajectory` input.
     """
-    return trajectory * norm_factor * (2 * resolution)
+    acq = acq or Acquisition.default
+    return trajectory * acq.norm_factor * (2 * acq.res)
 
 
 def unnormalize_trajectory(
     trajectory: NDArray,
-    norm_factor: float = KMAX,
-    resolution: float | NDArray = DEFAULT_RESOLUTION,
```

Comment:
Im not a fan of needing to provide all the acquisition config here. We should also be able to just specify the resolution rather. Right?